gslib/commands/publicaccessprevention.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we want to show this value for `gsutil ls -Lb gs://bucket` as well? I did not see any changes done for that.